pkgs/by-name/hu/humblebundle-downloader/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  fetchFromGitHub,
+  lib,
+  python3Packages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "humblebundle-downloader";
+  version = "0.4.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "xtream1101";
+    repo = "humblebundle-downloader";
+    tag = version;
+    hash = "sha256-fLfAGDKn6AWHJKsgQ0fBYdN6mGfZNrVs9n6Zo9VRgIY=";
+  };
+
+  build-system = with python3Packages; [
+    poetry-core
+  ];
+
+  dependencies = with python3Packages; [
+    parsel
+    requests
+  ];
+
```

Comment:
```suggestion
passthru.updateScript = nix-update-script { };
```

Reply:
`buildPythonApplication` already include an update script by default

Reply:
Good to know. Thanks.